apps/grader-host/src/index.js

```diff
@@ -39,15 +39,11 @@ process.on('SIGTERM', () => {
 
 async.series(
```

Comment:
At some point it would be great to start getting rid of all the `async.series()` calls and just having regular code calling `await ...` in series 😄

Reply:
Agreed! Two things standing partially in the way of that:

- We want ESM for top-level await (thought we could just do `async.series` with a single function for error handling at the top level)
- Some of the functions here are actually still callback-based - hopefully not for long!